### PR TITLE
Add tables for global QS indicators

### LINKED ISSUE #172: Extract general some Q&S indicators from antigen tab to General Q&S tab
Data elements to extract: 
 - ADS (Auto Disable Syringe) used
 - Safety Boxes.

No change in the Campaign Wizard. The user sets the general Q&S indicators per antigen. If the Q&S indicator is selected for at least a single antigen it will be displayed in **General Q&S** tab.

- [x] Modify metadata to allow declaring general indicators (not only per antigen)
- [x] Modify the form in data entry to show Q&S General indicators in a separate tab.
- [ ] Modify the dashboard: Show the general Q&S Indicators in a chart per organisation unit and team (no disaggregation per antigen) -> @matiasgarcia91

### PR DESCRIPTION
**Pull request by matiasgarcia91**
### :pushpin: References

* **Issue:** Closes #172

### :memo: Implementation
Add a table for the general QS indicators, not separated by antigen.

### Screenshot:
imagen

### :fire: Is there anything the reviewer should know to test it?
Still not tested, new metadata is needed: EyeSeeTea/vaccination-metadata@952f558

### CODE REVIEW

File: src/models/dashboard-items.js [hunk 1]
```diff
@@ -89,8 +99,16 @@ function getCharts(antigen, elements, organisationUnit, itemsMetadata, disaggreg
...
 
-function getTables(antigen, elements, organisationUnit, itemsMetadata, disaggregationMetadata) {
-    return _(dashboardItemsConfig.tables)
+function getTables(
+    antigen,
+    elements,
+    organisationUnit,
+    itemsMetadata,
+    disaggregationMetadata,
+    general = null
```
Comment (tokland) [inline src/models/dashboard-items.js]: It's a boolean,  right? then probably `general = false`. Also, having booleans as normal arguments is confusing when calling the function. I'd propse `..., disaggregationMetadata, {general = false} = {}`

File: src/models/dashboard-items.js [hunk 2]
```diff
@@ -468,7 +496,7 @@ const tableConstructor = ({
         dataElementGroupSetDimensions: [],
         attributeDimensions: [],
         translations: [],
-        filterDimensions: ["ou", antigenCategory],
+        filterDimensions: ["ou", antigen ? antigenCategory : null],
```
Comment (tokland) [inline src/models/dashboard-items.js]: It seems to work with null values in the array, but maybe it's safer to clean it with `_.compact`

File: src/models/dashboard-items.js (hunk 1 above)
Comment (matiasgarcia91) [reply src/models/dashboard-items.js]: yes it's boolean of course, my bad. Applied suggested changes

**Comment by adrianq** [comment]
As discussed with @matiasgarcia91 I am merging this PR although we will probably have to refactor this table as well:
image
It doesn't make too much to have these indicators per antigen when they are actually calculated like this:
safety boxes ratio = syringes for dilution / safety boxes
ADS wastage rate = ADS used - Total Doses adm / ADS used

Will create a card once discussed with MSF

**Pull request merged**